### PR TITLE
update the KubernetesSeedProvider.java in example example/cassandra to v1beta3

### PR DESCRIPTION
**Pull request by caesarxuchao**
update the KubernetesSeedProvider.java in example example/cassandra to v1beta3. This fixes issue #7949.
(I use #8023 as a reference.)

Could someone move the docker image from caesarxuchao/cassandra:latest to kubernetes/cassandra:v3? I don't have the write permission.

 @nikhiljindal , @pires, @thockin

### CODE REVIEW

File: examples/cassandra/java/src/io/k8s/cassandra/KubernetesSeedProvider.java [hunk 1]
```diff
@@ -64,18 +75,19 @@ public KubernetesSeedProvider(Map<String, String> params) {
...
-            if (endpoints.endpoints != null){
-		for (String endpoint : endpoints.endpoints) {
-		    String[] parts = endpoint.split(":");
-		    list.add(InetAddress.getByName(parts[0]));
+        if (endpoints != null) {
+            // Here is a problem point, endpoints.subsets can be null in first node cases.
+            if (endpoints.subsets != null && !endpoints.subsets.isEmpty()){
+                for (Subset subset : endpoints.subsets) {
+		for (Address address : subset.addresses) {
+            list.add(InetAddress.getByName(address.IP));
```
Comment (zmerlynn) [inline examples/cassandra/java/src/io/k8s/cassandra/KubernetesSeedProvider.java]: tab/space issue here?

Comment (caesarxuchao) [inline examples/cassandra/java/src/io/k8s/cassandra/KubernetesSeedProvider.java]: Thanks.

**Comment by nikhiljindal** [comment]
cc @brendandburns since he updated this last.

LGTM. Will wait for the docker image to be updated before merging.

**Comment by caesarxuchao** [comment]
@lavalamp , could you help move the docker image from caesarxuchao/cassandra:latest to kubernetes/cassandra:v3? I don't have the write permission. Thanks.

**Comment by lavalamp** [comment]
Should switch to gcr.io-- look at contrib/for-tests/network-tester's makefile for how to do it. I think you should be in the correct groups already.

(if you don't want to switch it over let me know.)

**Comment by caesarxuchao** [comment]
@nikhiljindal  I update the image registry. Let's wait for green.

**Comment by rjnagal** [comment]
Shippable green, merging.

**Pull request merged**